deepchem/models/atomic_conv.py

```diff
@@ -278,8 +177,6 @@ def __init__(
       layer = prev_layer
       if next_activation is not None:
         layer = Activation(next_activation)(layer)
-      print("size")
-      print(size)
       # layer = Dense(100)(layer)
```

Comment:
Should we go ahead and remove some of the commented lines here and elsewhere in this PR?

Reply:
Sounds good! I'll go ahead and remove the unnecessary comments.